openhands/cli/settings.py

```diff
@@ -158,8 +158,12 @@ async def modify_llm_settings_basic(
     provider_completer = FuzzyWordCompleter(provider_list)
     session = PromptSession(key_bindings=kb_cancel())
 
-    # Set default provider - use the first available provider from the list
-    provider = provider_list[0] if provider_list else 'openai'
+    # Set default provider - prefer 'anthropic' if available, otherwise use the first provider
+    provider = (
+        'anthropic'
+        if 'anthropic' in provider_list
+        else (provider_list[0] if provider_list else 'anthropic')
+    )
```

Comment:
```suggestion
    # Set default provider - prefer 'anthropic' if available, otherwise use the first provider
    provider = (
        'anthropic' if (provider_list is None or 'anthropic' in provider_list) else provider_list[0]
    )
```